migrations/versions/c3d3e960fc1b_sensitive_not_nullable.py

```diff
@@ -0,0 +1,39 @@
+"""sensitive_not_nullable
+
+Revision ID: c3d3e960fc1b
+Revises: a5a40d843415
+Create Date: 2022-02-19 10:34:15.584075
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "c3d3e960fc1b"
+down_revision = "a5a40d843415"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Seems like this migration does nothing?

Reply:
The autogenerate function probably couldn't detect what you did. I suggest you remove this file and add that requirement to the earlier migration

Reply:
> add that requirement to the earlier migration

How do I do that? 

Also I wasn't sure that setting it to nullable should be recognized but I guess that makes sense